koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -274,52 +291,45 @@ def parquet_file_getter(self):
             else get_s3_objects_not_matching_metadata
         )
 
-    def _set_post_processor(self):
+    @cached_property
+    def post_processor(self):
         """Post processor based on provider type."""
-        post_processor = None
-        if self.provider_type in [Provider.PROVIDER_AWS, Provider.PROVIDER_AWS_LOCAL]:
-            post_processor = AWSPostProcessor(schema=self._schema_name)
-        elif self.provider_type in [Provider.PROVIDER_AZURE, Provider.PROVIDER_AZURE_LOCAL]:
-            post_processor = AzurePostProcessor(schema=self.schema_name)
-        elif self.provider_type in [Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL]:
-            post_processor = GCPPostProcessor(schema=self._schema_name)
-        elif self.provider_type in [Provider.PROVIDER_OCI, Provider.PROVIDER_OCI_LOCAL]:
-            post_processor = OCIPostProcessor(schema=self._schema_name)
+        if self.provider_type == Provider.PROVIDER_AWS:
```

Comment:
The `self.provider_type` property strips the `-local` so we don't need to compare all of these with the `_LOCAL` provider types.